src/lwc/geTemplateBuilderTemplateInfo/geTemplateBuilderTemplateInfo.js

```diff
@@ -1,7 +1,20 @@
 import { LightningElement, api } from 'lwc';
 import { dispatch, isEmpty, isFunction } from 'c/utilTemplateBuilder';
 
+import geHeaderTemplateInfoLeftCol from '@salesforce/label/c.geHeaderTemplateInfoLeftCol';
```

Comment:
What's the reasoning for leaving these outside of geLabelService when the rest of the are handled there?

Reply:
Contrary to what we just talked about, these were actually missed :) Updating.